tests/utils/classes.inc

```diff
@@ -38,17 +39,18 @@ class Address implements MongoDB\BSON\Persistable {
     protected $zip;
     protected $country;
 
-    function __construct($zip, $country) {
+    public function __construct($zip, $country) {
         $this->zip = $zip;
         $this->country = $country;
     }
-    function bsonSerialize() {
+    #[\ReturnTypeWillChange]
```

Comment:
That is correct - PHP 8.1 adds the concept of tentative return types to alert users of upcoming return type changes (which have to be added downstream first). There's PHPC-1710 to document these type changes in interfaces, which I'll tackle once this PR is done. This will involve adding language to the interface documentation on php.net to point out this change and how to silence the errors without adding types first. In this case I didn't add a return type since it would be `mixed` which would require a test split due to the wide range of PHP versions we support. Users may be more flexible as to what return types they can add, especially as they can narrow the return type.